packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -532,6 +561,91 @@ Your $application code is in $relativeAppMain.
     return generatedCount;
   }
 
+  // TODO(dacoharkes): Can we remove duplication between plugin and ffPlugin?
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions